### PR TITLE
adding links

### PR DESCRIPTION
**Pull request by MBishop17**
added a gang of links to tutorials and k8's docs.

### CODE REVIEW

File: content/rancher/v2.x/en/concepts/resources/_index.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Within the context of a Rancher project or namespace, _resources_ are files and
 - [Secrets](#secrets)
 - [Registries](#registries)
 
-Rancher extends the application of the Kubernetes namespace resources listed above to [projects](../projects), which are Rancher-specific constructs. In the hierarchy of Rancher objects, projects contain namespaces. Therefore, any resources available within a project are available for all namespaces within that project.
+Rancher extends the application of the Kubernetes namespace resources listed above to [projects]({{< baseurl >}}/rancher/v2.x/en/projects), which are Rancher-specific constructs. In the hierarchy of Rancher objects, projects contain namespaces. Therefore, any resources available within a project are available for all namespaces within that project.
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/concepts/resources/_index.md]: not a valid link

File: content/rancher/v2.x/en/tasks/workloads/add-a-sidecar/_index.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 ---
 title: Adding a Sidecar
+<<<<<<< HEAD
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/tasks/workloads/add-a-sidecar/_index.md]: this needs to be cleaned up

File: content/rke/v0.1.x/en/config-options/add-ons/ingress-controllers/_index.md [hunk 3]
```diff
@@ -39,7 +39,11 @@ ingress:
 ```
 ## Configuring NGINX Ingress Controller
 
+<<<<<<< HEAD
```
Comment (deniseschannon) [inline content/rke/v0.1.x/en/config-options/add-ons/ingress-controllers/_index.md]: same with this

**Comment by MBishop17** [comment]
I think I finally fixed this.

**Pull request merged**